### PR TITLE
Remaining troubleshooting files and other content

### PR DESCRIPTION
**Pull request by scottcwilson**

### CODE REVIEW

**Comment by scottcwilson** [comment]
This should be the last big update.

File: content/user/new_user_topics/no_such_file.md [hunk 1]
```diff
@@ -1,24 +1,26 @@
 ---
-title: I don't have the file XXXXX
-description: Zen Cart File missing 
+title: I don't have the file YOURTEMPLATE/some-name
+description: Zen Cart Templated File missing
```
Comment (drbyte) [inline content/user/new_user_topics/no_such_file.md]: Templated or Template?

File: content/user/new_user_topics/no_such_file.md [hunk 2]
```diff
@@ -34,13 +36,41 @@ this file would be `includes/templates/responsive_classic/templates/tpl_product_
...
 
-So `includes/modules/responsive_sheffield_blue/additional_images.php` is a templated file based on `includes/modules/additional_images.php`.
+Steps: 
+- Using FTP, connect to `YOURSITE.com`.  
+- Change directories so you get to the webroot.  
+- Change directories from there so that you get to the folder for the file you are changing.  At this point, you will be in `/home/johndoe/public_html/includes/languages/custom`
+- Transfer the file.
 
-Likewise, `includes/languages/responsive_classic/english.php` is a templated modification of `includes/languages/english.php`.
+Here's what it looks like for my store.  Click the image to make it larger if needed.  As you can see, my template is `bootstrap` and my webroot is `/home/thatsoft/public_html`.
```
Comment (drbyte) [inline content/user/new_user_topics/no_such_file.md]: Did you want this to be personalized this heavily? "my store", "my template", "thatsoft"

File: content/user/payment/paypal_troubleshooting.md [hunk 3]
```diff
@@ -32,7 +32,7 @@ These are the common configuration errors causing IPN processing to fail:
 5.  Use your browser and go to the Login page of your store. If the page is SSL, do you get any certificate errors in your browser? Test from a couple different computers that you don't normally use. An invalid SSL certificate or one that has errors of any sort, could prevent PayPal from successfully posting the notices to your site.
 6.  Try accessing `https://YOURSITE.com/ipn_main_handler.php` with your browser. If you see PHP errors, those will need to be resolved. If you get a white screen, then the first phase of PayPal™ contacting your site isn't encountering errors. This doesn't mean there aren't any, it just means the initial most obvious steps are working.
 7.  There are [two communication testing tools available in the support forum](http://www.zen-cart.com/forum/showthread.php?t=65680). If you are asking for troubleshooting help, please supply the URL to each of these tools after you have installed them on your site, so we can assess the responses they reveal.
```
Comment (drbyte) [inline content/user/payment/paypal_troubleshooting.md]: We should probably update all the `http:` links to `zen-cart.com` to `https`

File: content/user/new_user_topics/no_such_file.md (hunk 1 above)
Comment (scottcwilson) [reply content/user/new_user_topics/no_such_file.md]: I was wavering.  Will fix.

File: content/user/new_user_topics/no_such_file.md (hunk 2 above)
Comment (scottcwilson) [reply content/user/new_user_topics/no_such_file.md]: The trouble is I wanted it to match the image.   But I can certainly eliminate the "my" usage.

File: content/user/payment/paypal_troubleshooting.md (hunk 3 above)
Comment (drbyte) [reply content/user/payment/paypal_troubleshooting.md]: Preferably as a separate PR

File: content/user/new_user_topics/no_such_file.md (hunk 2 above)
Comment (drbyte) [reply content/user/new_user_topics/no_such_file.md]: I'm not bothered by it. Just wanted to be sure you meant it.

File: content/user/payment/paypal_troubleshooting.md (hunk 3 above)
Comment (scottcwilson) [reply content/user/payment/paypal_troubleshooting.md]: done

Comment (scottcwilson) [reply content/user/payment/paypal_troubleshooting.md]: oops! sorry.

**Pull request merged**